inc/composer/class-command.php

```diff
@@ -971,15 +983,22 @@ protected function ssl( InputInterface $input, OutputInterface $output ) {
 				break;
 
 			case '':
-				$cert_exists = file_exists( 'vendor/ssl-cert.pem' ) && file_exists( 'vendor/ssl-key.pem' );
+				$cert_file = 'vendor/ssl-cert.pem';
+				$cert_exists = file_exists( $cert_file ) && file_exists( $cert_file );
 				if ( ! $cert_exists ) {
 					$output->writeln( "<error>Certificate file does not exist. Use 'composer server ssl generate' to generate one. </error>" );
 					return 1;
-				} else {
-					$output->writeln( '<info>Certificate file exists.</info>' );
 				}
 
-				break;
+				// Check certificate expiration.
+				$validity_remaining = $this->check_ssl_expiry( $cert_file );
```

Comment:
Hmm are we catching this error? Do we want to block being able to start the server if the `openssl` call fails? IMO best to just show a warning "Could not get certificate expiry details" but still start the server?